components/Pagination/sub-components/Mobile.jsx

```diff
@@ -86,6 +86,7 @@ const Mobile = ({
       href={handleHref(activePage - 1)}
       rightSquared
       theme={theme}
+      rel={activePage > 2 ? 'nofollow' : ''}
```

Comment:
shouldn't it also be `1` here?

Reply:
This is the prev button. If `activePage` is 3 or more it should have `nofollow` also.